README.md

```diff
@@ -87,11 +87,20 @@ Accessibility should be incorporated and tested throughout the design and develo
 
 ## Policy & Informal
 
-* **Analytics:** All of our sites should have code snippets for 18F and Digital Analytics Program Google Analytics as well as Google Webmaster Tools.
 * Agency or Organization Communications Office
 * Legislative Affairs
 * Office of General Counsel
 
+### Analytics
+
+All of our sites should have code snippets for
+
+* 18F Google Analytics (18F only)
+* Digital Analytics Program (DAP)
+* Google Webmaster Tools
```

Comment:
IMHO, I think that we haven't used it much. However, I defer to @gbinal to officially advise on GWT, b/c he has a greater understanding.